alerts/openshift-container-storage-operator/check-ceph-network-connectivity.md

```diff
@@ -0,0 +1,88 @@
+
```

Comment:
I think this file is already included in the helpers folder as "networkConnectivity.md". Please remove this and us the one already present

Reply:
Yes, you are right. Done